src/app/data/DatabaseConnection.java

```diff
@@ -30,6 +30,7 @@ public void connectToDatabase() {
             System.out.println("Unsuccessful connection to the database");
             e.printStackTrace();
         } catch (ClassNotFoundException e){
```

Comment:
This catch should be collapsed into a single catch as the contents of the catch are identical to the previous